src/style.css

```diff
@@ -2,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
It's recommended to include a fallback font in your `font-family` to ensure proper rendering if the primary font fails to load. You have included Arial and sans-serif as fallbacks, which is good practice.